### ISSUE TITLE
salter harris fractures

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10707]
**Preferred term label:**
salter harris I fracture
salter harris II fracture
salter harris III fracture
salter harris IV fracture

**Synonyms**
Respectively:
Physeal separation
Fracture starts in growth plate and exits through metaphysis
Fracture that traverses the physis and exits through the epiphysis
Fracture that passes through epiphysis, physis, and metaphysis in a child

**Definition (free text, please give PubMed ID)**
See https://radiopaedia.org/articles/salter-harris-classification?lang=us

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
I: Seymour fracture
II: Seymour fracture
II, IV: not yet added by our orthopedist

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

according to this there are five kinds (not IV):
https://www.ncbi.nlm.nih.gov/books/NBK430688/
Should we make one term for each kind?

**Comment by MickeySegal** [comment]
It is fine to make all 5.  Type V is said to be rare, so it may just not have come up yet in curation.

**Comment by pnrobinson** [comment]
@MickeySegal 
adding the following as a superclass for the five new terms
Adding new term: 
- ID: HP:6001243
- label: Physeal fracture
- parents: [Fracture type (HP:4000042)](https://hpo.jax.org/app/browse/term/HP:4000042)
- pmid: PMID:28613461
- def: Salter-Harris fractures (physeal fractures) refer to fractures through a growth plate (physis) and are, therefore, specifically applied to bone fractures in children.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10707
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal note that the parent for this and the other terms will be the above one, I need to transfer it in Protege
Adding new term: 
- ID: HP:6001244
- label: Salter I fracture
- parents: [Fracture type (HP:4000042)](https://hpo.jax.org/app/browse/term/HP:4000042)
- pmid: PMID:28613461
- def: A type of physeal (Salter-Harris) fracture in which the fracture line extends through the physis or within the growth plate. Type I fractures are due to the longitudinal force applied through the physis, which splits the epiphysis from the metaphysis.
- comment: Beware that a normal radiograph cannot exclude a physis injury in a symptomatic pediatric patient. A radiograph may be normal due to lack of bony involvement, and mild to moderate soft tissue swelling may be noted. Look for the widening of the physis or displacement of the epiphysis, which may suggest a fracture. Diagnosis is based on clinical findings, such as the presence of focal tenderness or swelling surrounding the growth plate.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10707
- synonyms: Type I Salter-Harris fracture(NONE)|Slipped physeal fracture(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001245
- label: Salter II fracture
- parents: [Fracture type (HP:4000042)](https://hpo.jax.org/app/browse/term/HP:4000042)
- pmid: PMID:28613461
- def: A type of physeal (Salter-Harris) fracture in which the fracture extends through both the physis and metaphysis. These are most common and occur away from the joint space. When the small corner of the metaphysis is visible, this is known as a corner sign or Thurston-Holland fragment.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10707
- synonyms: Type II Salter-Harris fracture(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001246
- label: Salter III fracture
- parents: [Fracture type (HP:4000042)](https://hpo.jax.org/app/browse/term/HP:4000042)
- pmid: PMID:28613461
- def: A type of physeal (Salter-Harris) fracture in which the fracture is intra-articular and extends from the physis into the epiphysis.
- comment: If the fracture extends the complete length of the physis, this type of fracture may form two epiphyseal segments. Since the epiphysis is involved, damage to the articular cartilage may occur. One example is a Tillaux fracture of the ankle, which is a fracture of the anterolateral aspect of the growth plate and epiphysis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10707
- synonyms: Type III Salter-Harris fracture(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001247
- label: Salter IV fracture
- parents: [Fracture type (HP:4000042)](https://hpo.jax.org/app/browse/term/HP:4000042)
- pmid: PMID:28613461
- def: A type of physeal (Salter-Harris) fracture in which the fracture is intra-articular and passes through the epiphysis, physis, and metaphysis.
- comment: As this fracture involves the epiphysis, the articular cartilage may be damaged. Types III and IV fractures each carry a risk for growth retardation, altered joint mechanics, and functional impairment. Therefore, both require urgent orthopedic evaluation.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10707
- synonyms: Type IV Salter-Harris fracture(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001248
- label: Salter V fracture
- parents: [Fracture type (HP:4000042)](https://hpo.jax.org/app/browse/term/HP:4000042)
- pmid: PMID:28613461
- def: A type of physeal (Salter-Harris) fracture that is due to a crush or compression injury of the growth plate. In type V, the force is transmitted through the epiphysis and physis, potentially disrupting the germinal matrix, hypertrophic region, and vascular supply.
- comment: Though Harris-Salter V fractures are very rare, they may be seen in electric shock, frostbite, and irradiation cases. As this fracture pattern tends to result from severe injury, these typically have a poor prognosis leading to bone growth arrest.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10707
- synonyms: Type V Salter-Harris fracture(NONE)
